### ISSUE TITLE
image size

### ISSUE THREAD

**Issue by abdallah1989203** [ultralytics/ultralytics#3275]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

from PIL import Image

img = Image.open('/val/images/img0.PNG') 

print(img.size)
-----------------------------------------------------------------------------------------------------------------------
(1920, 1080)

# Train the model YOlov8
model.train(data='coco128.yaml', epochs=100, imgsz=640)

I have images with a resolution of (1920, 1080) and the imgsz for YOLOv8 is set to 640. How can I resize the images without losing any information? Could I just use the resolution (1920, 1080) directly? Or is there a better solution?
can anyone help?

**Comment by Laughing-q** [comment]
@abdallah1989203 don't worry, resize operation in yolov8 keeps the aspect ratio of original image, so you won't lose any information except the longest image size become 640 instead of 1920.
For training your custom dataset, you can always start with the default training settings, try to train it with imgsz=640(except the objects you want to detect in your images are very small, this case you might want a higher imgsz). If the result does not look great then you can try to modify some hyps.

**Comment by liuhuan-gl** [comment]
@abdallah1989203 对，楼上说的没问题，我分别试过修改与不修改，几乎记过一样

**Comment by glenn-jocher** [comment]
Hi @liuhuan-gl, 

Yes, you can directly use the resolution (1920, 1080) as the input size for your YOLOv8 model. However, keep in mind that YOLOv8 expects a square input image, so you will need to resize your images to a square shape. 

When resizing the images to a square shape, you may lose some information from the original image, especially if the aspect ratio of the original image is significantly different from a square. To minimize information loss, you can try to maintain the aspect ratio of the original image by padding the resized image with zero values.

Alternatively, you can set the `imgsz` parameter in the YOLOv8 training command to a square size, such as 640x640, which is the default value. This will ensure that all input images are resized to a square shape without losing too much information.

I hope this helps! If you have any further questions, feel free to ask.
Best regards,
Glenn Jocher
